shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -576,6 +576,7 @@
 void FlutterPlatformViewsController::BringLayersIntoView(LayersMap layer_map) {
   UIView* flutter_view = flutter_view_.get();
   auto zIndex = 0;
+  active_composition_order_.clear();
```

Comment:
could you add a comment? something like: "Clear the active composition order since the new composition order is determined down below".

Reply:
done